src/streamlink/plugins/mdstrm.py

```diff
@@ -0,0 +1,140 @@
+"""
+$description CDN hosting live content for various websites in the Americas.
+$url mdstrm.com
+$type live
+"""
```

Comment:
Please add all the supported URLs to the plugin metadata

```suggestion
"""
$description CDN hosting live content for various websites in the Americas.
$url mdstrm.com
$url latina.pe/tvenvivo
$url saltillo.multimedios.com/video
$type live
"""
```